docs/content/latest/api/ysql/datatypes/type_datetime/stopwatch.md

```diff
@@ -12,11 +12,11 @@ isTocNested: true
 showAsideToc: true
 ---
 
-You sometimes want to time a sequence of several SQL statements, issued from _ysqlsh_, and to record the time in the spool file. The _\timing on_ metacommand doesn't help here because it reports the time after every individual statement and, on Unix-like operating systems, does this using _stderr_. The _\o_ metacommand doesn't redirect _stderr_ to the spool file. This case study shows you how to implement a SQL stopwatch that allows you to start it with a procedure call before starting what you want to time and to read it with a _select_ statement when what you want to time finishes. This reading goes to the spool file along with all other _select_ results.
+You sometimes want to time a sequence of several SQL statements, issued from _ysqlsh_, and to record the time in the spool file. The &#92;_timing on_ metacommand doesn't help here because it reports the time after every individual statement and, on Unix-like operating systems, does this using _stderr_. The &#92;_o_ metacommand doesn't redirect _stderr_ to the spool file. This case study shows you how to implement a SQL stopwatch that allows you to start it with a procedure call before starting what you want to time and to read it with a _select_ statement when what you want to time finishes. This reading goes to the spool file along with all other _select_ results.
```

Comment:
Please don't use HTML entities!

Reply:
I replaced all occurrences of `&#92;` with `//`. This was an easy fix. That approach hadn't occurred to me before. The global fix changed thirteen files. Also:

- I'd used `&#91;` and `&#93;` for `[` and `]` inside the text of a link which Markdown syntax requires surrounding with its own `[` and `]`. In some (but by no means all) cases, having unescaped `[` and `]` inside the Markdown's own `[` and `]` produced results with badly broken formatting. It's far too hard to analyze each of these as a special case. So I'd adopted a global solution. I just changed the global approach use `\[` and `\]` inside link text.

- I'd used `&#60;` and `&#62;` for `<` and `>` (for obvious reasons). I changed these globally to `\<` and `\>`

- I'd used `&#124;` for `|` because, inside table cells, it simply broke the table. I replaced these with `\|` globally.

- I'd used `&#42;` for `*` because it was sometimes taken as the Markdown directive for italic. I replaced these with `\*` globally.